packages/zero-schema/src/permissions.ts

```diff
@@ -16,8 +16,13 @@ import type {
 } from './compiled-permissions.ts';
 import {clientToServer, NameMapper} from './name-mapper.ts';
 
-export const ANYONE_CAN = undefined;
+export const ANYONE_CAN = [
+  (authData: {sub: string}, eb: ExpressionBuilder<Schema, never>) =>
+    // TODO: Could just add a true() predicate?
+    eb.cmpLit(authData.sub, 'NOT IN', []),
```

Comment:
nit: `eb.cmpLit(true, true)` will work

```
npm run run-query -- --debug-query 'z.query.issue.where(({cmpLit}) => cmpLit(false, "=", true))'
> zbugs@0.0.0 run-query
> npm run zero-build-schema && tsx ../../packages/zero-cache/src/scripts/run-query.ts --debug-query z.query.issue.where(({cmpLit}) => cmpLit(false, "=", true))


> zbugs@0.0.0 zero-build-schema
> tsx ../../packages/zero-schema/src/build-schema.ts

=== Query Stats: ===

issue vended: []
total rows considered: 0
time: 1.45ms ms


=== Query Plans: ===
```

Reply:
would `eb.and()` work?

I tried running npm run run-query to check but it does not work for me:

<img width="1132" alt="image" src="https://github.com/user-attachments/assets/b447fc27-d5f6-4a70-a407-d4a28b14e141" />

Reply:
I think that would work too.

That's really strange that `where` is undefined for you. Maybe it has to do with the updates that remove `schema.json`? Since the schema is required for `run-query` to create the query interface.

Reply:
Done! Super elegant, I love it. And I used `or([])` for `false`.